packages/router/src/events.ts

```diff
@@ -59,6 +60,9 @@ export class NavigationStart extends RouterEvent {
    * Identifies the call or event that triggered the navigation.
    * An `imperative` trigger is a call to `router.navigateByUrl()` or `router.navigate()`.
    *
+   * @see `NavigationEnd`
+   * @see `NavigationCancel`
+   * @see `NavigationError`   *
```

Comment:
```suggestion
   * @see `NavigationError`
   * @see `NavigationStart`
```

Based on the other updates, it looks like you're prioritizing links to these 4 events?